### PR TITLE
#313 | Header language switcher

### LINKED ISSUE #313: Header internationalization
## Overview
The new header needs to include the language switcher and be translated.

## Acceptance criteria
- add i18n for header text
- Design language switcher
- Implement language switcher

## Tasks
- [x] add i18n to existing header text
- [x] add language switcher for mobile and desktop
- [x] add i18n for new language switcher
- [ ] create ticket to update portuguese translations to match https://github.com/telosnetwork/teloscan/pull/318

### PR DESCRIPTION
**Pull request by EJSG**
# Fixes #313

## Description
This PR adds a language switcher to the header, as well as fixes a couple missing translations

Big thanks to @Viterbo for the js to make the language switcher work
 

## Test scenarios
**Broken translations**
- log in
- go to https://deploy-preview-318--teloscan-stage.netlify.app/staking
- enter any amount into the staking form
- click "Stake"
    - all localizations in the confirmation modal should look good
- go to https://deploy-preview-318--teloscan-stage.netlify.app/staking#unstake
- enter any amount into the unstake form
- click "Unstake"
    - all localizations in the confirmation modal should look good

**Language switcher**
- on both mobile and desktop:
- click Advanced on the site header
- click Language
    - a language switcher modal should come up 
- select a different language
    - app should reload with new language 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/i18n/en-us/index.js [hunk 1]
```diff
@@ -300,13 +302,11 @@ export default {
             write: 'Write',
             amount: 'Amount',
             value: 'Value',
-            ok: 'Ok',
```
Comment (EJSG) [inline src/i18n/en-us/index.js:303]: moved 'ok' and 'cancel' to namespace `global`

File: src/i18n/en-us/index.js [hunk 2]
```diff
@@ -130,7 +133,6 @@ export default {
         paste_contract_code_here: 'copy & paste contract code here...',
         enter_contract_text: 'enter or paste contract text',
         verify_contract: 'Verify Contract',
-        dismiss: 'Dismiss',
```
Comment (EJSG) [inline src/i18n/en-us/index.js:133]: moved 'dismiss' to namespace `global`

File: src/pages/staking/UnstakeForm.vue [hunk 3]
```diff
@@ -35,22 +35,22 @@
...
-                    
+
                 </p>
                 <p v-if="remainingDeposits < 10">
                     {{ $t('pages.staking.confirm_unstake_2a') }}
                     <span class="text-primary">{{ remainingDeposits }}</span>
                     {{ $t('pages.staking.confirm_unstake_2b') }}
 
                 </p>
-                {{ $t('pages.staking.confirm_unstake_3') }}
```
Comment (EJSG) [inline src/pages/staking/UnstakeForm.vue:46]: fix broken slug

**Review by donnyquixotic — APPROVED** [review]

File: src/i18n/en-us/index.js (hunk 2 above)
Comment (Viterbo) [reply src/i18n/en-us/index.js:133]: Ok. This will require replicating the same index.js refactor in the incoming Portuguese PR.
I'll take care of it

**Review by Viterbo — APPROVED** [review]

**Pull request merged**